docs/source/install.rst

```diff
@@ -44,6 +44,7 @@ credentials to log in to Django admin.
 You will only need to run the following two commands the next time you want to
```

Comment:
two -> four

Could you also add

```sh
$ . venv/bin/activate
```

to the command list? I forgot to add it.

Reply:
Ready, I add two more command. 1. ```cd ~/pythondotorg``` and 2. ```. venv/bin/activate```